libpod/runtime.go

```diff
@@ -233,6 +237,8 @@ type RuntimeConfig struct {
 	EventsLogger string `toml:"events_logger"`
 	// EventsLogFilePath is where the events log is stored.
 	EventsLogFilePath string `toml:-"events_logfile_path"`
```

Comment:
Completely unrelated to this PR, but what is this `-` doing in the TOML string... @baude you added this, right?